bvm/ballerina-config/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
     includedJarLibs 'com.moandjiezana.toml:toml4j'
     includedJarLibs 'org.wso2.orbit.org.antlr:antlr4-runtime'
     includedJarLibs 'org.slf4j:slf4j-api'
+    includedJarLibs 'org.slf4j:slf4j-simple'
```

Comment:
This binding is not the one we use. We use `slf4j-jdk14` binding. All other bindings should be excluded.

Reply:
Fix with correct binding. Thanks